README.md

```diff
@@ -249,6 +249,22 @@ let t = of_json (Json.parseOrRaise {|{"a": 42}|})
 (* t = { a = 42; b = None; } *)
 ```
 
+#### `[@json.drop_default]`: drop default values from JSON
+
+When a field has `[@option]` attribute one can use `[@json.drop_default]`
```

Comment:
What happens now if we use `drop_default` in a non option field? Also, would it be worth to test this case? To have it documented "internally".

Reply:
added a test case